internal/symbolizer/symbolizer.go

```diff
@@ -121,7 +121,7 @@ func postURL(source, post string) ([]byte, error) {
 	}
 	defer resp.Body.Close()
 	if resp.StatusCode != http.StatusOK {
```

Comment:
Optional: this error code path doesn't seem to be tested. It's not quite fair to ask you to add missing tests but if you will like it, it would be welcome.